simdepot.h

```diff
@@ -102,6 +102,12 @@ class depot_t : public gebaeude_t
 	 * @param local_execution if true, this method creates pop-ups in case of errors
 	 */
 	bool start_convoi(convoihandle_t cnv, bool local_execution);
+	/**
+	 * This is only for coupled couvoy
+	 * only return the condition of leaving, 
+	 * coupled convoy does not depart.
+	 */
+	bool can_start_coupled_convoi(convoihandle_t cnv, bool local_execution); 
```

Comment:
この関数定義だけだと引数の `cnv` が親編成なのか小編成なのかがわかりにくいので、コメントと関数名に工夫が必要です。関数名は `can_start_when_coupled` でしょうか。
```suggestion
	/**
	 * This is only for coupled couvoy
	 * only return the condition of leaving, 
	 * coupled convoy does not depart.
	 */
	bool can_start_when_coupled(convoihandle_t cnv, bool local_execution); 
```

Reply:
ここについては、start_convoi()の発車条件確認部分と全く同じ処理であるため、この関数を
```bool can_start_convoi()```
に改め、```start_convoi()```内部で```can_start_convoi()```を呼び出し、```true```のときのみ```cnv->start()```としたいと思います。